### PR TITLE
fix distance

### PR DESCRIPTION
**Pull request by LIUSHANYIN**

### CODE REVIEW

File: src/components/Stepper/index.scss [hunk 1]
```diff
@@ -174,3 +180,7 @@ input.#{$baseClass} {
   margin: auto;
   opacity: 0;
 }
+.load {
+  position: absolute;
+  bottom: 150px;
```
Comment (mxdi9i7) [inline src/components/Stepper/index.scss:185]: are you sure this bottom value is correct?

File: src/components/Stepper/index.tsx [hunk 2]
```diff
@@ -30,9 +32,10 @@ export default function Stepper({
   disableInput,
   size,
   theme,
-  loading
+  loading,
+  onChange
 }: IProps) {
-  const [value, setValue] = useState(0);
+  let [value, setValue] = useState(0);
```
Comment (mxdi9i7) [inline src/components/Stepper/index.tsx:38]: why using let?

File: src/components/Stepper/index.scss (hunk 1 above)
Comment (LIUSHANYIN) [reply src/components/Stepper/index.scss:185]: yes, I am sure.

File: src/components/Stepper/index.tsx (hunk 2 above)
Comment (LIUSHANYIN) [reply src/components/Stepper/index.tsx:38]: Because I need to pass the value as parameter and I need value+ and value-.

**Pull request merged**